setup.py

```diff
@@ -13,7 +13,8 @@
     'console_scripts': [
         'insights-run = insights.core:main',
         'gen_api = insights.tools.generate_api_config:main',
-        'insights-perf = insights.tools.perf:main'
+        'insights-perf = insights.tools.perf:main',
+        'client = insights.client:run'
```

Comment:
Does this mean we can now invoke the core and client with args such as 'insights-core client --args1 --args2 --etc'?